src/views/Player.vue

```diff
@@ -134,6 +135,12 @@ export default {
       tutorialProgress: {},
       isTutorialUploadRequired: false,
       isModalOnScreen: false,
+      componentProperties: {},
```

Comment:
a data property to store the `component-properties.json`, i.e. the system wide properties we (might) have when we decide to keep a feature always

Reply:
This needs defaults that will be populated from the `defaults` json file